packages/mdl-snackbar/mdl-snackbar.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/mixins";
+@import "mdl-theme/mixins";
+@import "mdl-typography/mixins";
+@import "./variables";
+
+/* postcss-bem-linter: define snackbar */
+.mdl-snackbar {
+  display: block;
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  justify-content: space-between;
+  transform: translate(0, 80px);
+  transition: transform .25s $mdl-animation-fast-out-linear-in-timing-function;
+  background-color: $snackbar-background-color;
+  cursor: default;
+  z-index: 3;
+  will-change: transform;
+  pointer-events: none;
+
+  @media (max-width: ($snackbar-tablet-breakpoint - 1)) {
+    left: 0;
+    width: 100%;
+    min-height: 48px;
+    max-height: 80px;
+  }
+
+  @media (min-width: $snackbar-tablet-breakpoint) {
+    min-width: 288px;
+    max-width: 568px;
+    transform: translate(-50%, 80px);
+    border-radius: 2px;
+  }
+
+  &--active {
+    transform: translate(0, 0);
+    pointer-events: auto;
+    transition: transform .25s $mdl-animation-linear-out-slow-in-timing-function;
+
+    @media (min-width: $snackbar-tablet-breakpoint) {
+      transform: translate(-50%, 0);
+    }
+  }
+  &__text {
```

Comment:
Undoing the float stuff SGTM. It was a fallback for IE9 only which then restricted what else I could do originally to get just close :tm:. Now we don't have that limitation so there isn't anything stopping it from being done properly in whole. :smile: